nomad/structs/structs.go

```diff
@@ -1153,6 +1154,9 @@ type Node struct {
 	// updated
 	StatusUpdatedAt int64
 
+	// NodeEvents is a list of the last 10 or lest events for this node
```

Comment:
`NodeEvents is the most recent set of events generated for the node.`
Spelling error and generally don't include the const values in comments. Keep it a bit more generic